File: src/score/ScoreClient.php
<?php

namespace telesign\enterprise\sdk\score;

use telesign\sdk\score\ScoreClient as _ScoreClient;

/**
 * Score provides risk information about a specified phone number,
 */
class ScoreClient extends _ScoreClient {

  function __construct ($customer_id, $api_key, $rest_endpoint = "https://rest-ww.telesign.com", ...$other) {
    parent::__construct($customer_id, $api_key, $rest_endpoint, ...$other);
  }

}

File: src/main/java/com/telesign/enterprise/ScoreClient.java
package com.telesign.enterprise;

import java.net.Proxy;

public class ScoreClient extends com.telesign.ScoreClient {

    public ScoreClient(String customerId, String apiKey) {

        super(customerId, apiKey, "https://rest-ww.telesign.com");
    }

    public ScoreClient(String customerId, String apiKey, String restEndpoint) {
        super(customerId, apiKey, restEndpoint);
    }

    public ScoreClient(String customerId,
                         String apiKey,
                         String restEndpoint,
                         Integer connectTimeout,
                         Integer readTimeout,
                         Integer writeTimeout,
                         Proxy proxy,
                         final String proxyUsername,
                         final String proxyPassword) {
        super(customerId, apiKey, restEndpoint, connectTimeout, readTimeout, writeTimeout, proxy, proxyUsername, proxyPassword);
    }

}
